sdcm/utils/toppartition_util.py

```diff
@@ -0,0 +1,144 @@
+import random
+import logging
+import re
+
+
+from abc import abstractmethod, ABC
+from collections import OrderedDict
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TopPartitionCmd(ABC):
+
+    def __init__(self, ks_cf_list: dict):
+        self.ks_cf_list: list = ks_cf_list
+        self._built_args: dict = {}
+        self._generate_random_parameters_values()
```

Comment:
Move this out to the disrupt_show_toppartitions and make it public.

Reply:
fixed